CHANGES

```diff
@@ -39,6 +39,10 @@ Features added
 Bugs fixed
 ----------
 
+* #11418: Clean up remaining references to ``sphinx.setup_command``
+  following the removal of support for setuptools.
+  Patch by Willem.
```

Comment:
@14mRh4X0r please do update with your surname if you'd like, I only saw your first name on your profile.